packages/flutter_tools/test/general.shard/asset_bundle_package_fonts_test.dart

```diff
@@ -48,12 +47,6 @@ $fontsSection
 ''');
   }
 
-  void establishFlutterRoot() {
```

Comment:
All of this removed code is a no-op now that the material manifest is stored in code